download.py

```diff
@@ -0,0 +1,185 @@
+"""_summary_
```

Comment:
Suggest adding your name/github username here as author, and adding MIT license.

Reply:
Good thinking.